modules/standard/BigInteger.chpl

```diff
@@ -291,6 +291,7 @@ module BigInteger {
       }
     }
 
+    deprecated "bigint.size() is deprecated"
```

Comment:
Should this suggest to use another method like `sizeInBase()` or anything like that? Or is it that we truly just don't want to support this functionality anymore?

Reply:
`size` returns "size in number of limbs", which is a notion from the underlying GMP implementation, while "sizeInBase" returns the number of digits used to represent the number in the provided base system.  I could redirect to numLimbs, but it sounds like we want to deprecate that method as well, so I chose the path that was least likely to lead to users having to update their code multiple times.

Reply:
Sounds good, thanks for clarifying